app/routes/rentals/show.js

```diff
@@ -0,0 +1,11 @@
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+  model(params) {
+    return this.store.queryRecord('rental', { slug: params.slug });
+  },
+
+  setupController(controller, model) {
```

Comment:
My opinion on `setupController` is while I don't have anything against it, I think that its something extra that we don't need to introduce/explain in the beginner's tutorial.  I'd prefer to refer to the model as `model`, especially with the uncertain future for the controller.